cplusplus.json

```diff
@@ -25,5 +25,119 @@
+  "imports": ["cntk"], "examples": ["https://github.com/Microsoft/CNTK/tree/master/Examples"]},
+  {"id": "cpp.tiny-dnn", "name": "tiny-dnn", "tech": ["dl"], "tags": ["framework"], "repo": "tiny-dnn/tiny-dnn",
+  "imports": ["tiny_dnn/tiny_dnn.h"], "examples": ["https://github.com/tiny-dnn/tiny-dnn/tree/master/examples"]},
+  {"id": "cpp.veles", "name": "veles", "tech": ["dl"], "tags": ["framework"], "repo": "Samsung/veles",
+  "imports": ["veles.h"], "examples": ["https://github.com/Samsung/veles/tree/master/libVeles/tests"]},
+  {"id": "cpp.kaldi", "name": "Kaldi", "tech": ["speech-recognition"], "tags": ["toolkit"], "repo": "kaldi-asr/kaldi",
+  "imports": ["base/kaldi-common.h"], "examples": ["https://github.com/kaldi-asr/kaldi/tree/master/egs"]},
+  
+  {"id": "cpp.asio", "name": "Asio", "tech":["networking"], "tags":["async-io", "async-networking"], "repo": "chriskohlhoff/asio",
+  "imports": ["asio.hpp"], "examples": ["https://github.com/chriskohlhoff/asio/tree/master/asio/src/examples"]},
+  {"id": "cpp.boost-asio", "name": "Boost.Asio", "tech":["networking"], "tags":["async-io", "async-networking"], "repo": "chriskohlhoff/asio",
+  "imports": ["asio.hpp"], "examples": ["https://github.com/chriskohlhoff/asio/tree/master/asio/src/examples", "http://think-async.com/"]},
+  {"id": "cpp.actor-framework", "name": "Actor Framework", "tags": ["concurrency", "framework"], "tech": ["faster-code"], "repo": "actor-framework/actor-framework",
+  "imports": ["caf/all.hpp"], "examples": ["https://github.com/actor-framework/actor-framework/tree/master/examples"]},
+  {"id": "cpp.libev", "name": "libev", "tags":["event", "async"], "tech": ["networking"], "repo": "http://software.schmorp.de/pkg/libev.html",
+  "imports": ["ev.h", "ev++.h"], "examples": ["http://pod.tst.eu/http://cvs.schmorp.de/libev/ev.pod"]},
+  {"id": "cpp.libevent", "name": "libevent", "tags":["event", "async"], "tech": ["networking"], "repo": "libevent/libevent",
+  "imports": ["event2/event.h", "event2/dns.h", "event2/util.h", "event2/listener.h", "event2/buffer.h", "event2/bufferevent.h"],
+  "examples": ["https://github.com/libevent/libevent/tree/master/sample"]},
+  {"id": "cpp.libuv", "name": "libuv", "tech":["networking"], "tags":["async-io"], "repo": "libuv/libuv",
+  "imports": ["uv.h"], "examples": ["https://github.com/libuv/libuv/tree/v1.x/samples"]},
+  {"id": "cpp.uvw", "name": "uvw", "tech":["networking"], "tags":["async-io"], "repo": "skypjack/uvw",
+  "imports": ["uvw.hpp"], "examples": ["https://github.com/skypjack/uvw#code-example"]},
+
+  {"id": "cpp.kfr", "name": "kfr", "tech": ["computational-science"], "tags": ["dsp", "audio", "fft"], "repo": "kfrlib/kfr",
+  "imports": ["kfr/base.hpp", "kfr/dft.hpp", "kfr/dsp.hpp", "kfr/io.hpp"], "examples": ["https://github.com/kfrlib/kfr/tree/master/examples"]},
+  {"id": "cpp.maximilian", "name": "Maximilian", "tech": ["computational-science"], "tags": ["dsp", "audio", "fft"], "repo": "micknoise/Maximilian",
```

Comment:
should this be tech == audio? 

computational science is pretty generic, and we like to reserve it for things like optimization algos, math, etc.

Reply:
kfr and Maximilian seems to provide pure dsp and fft related algorithms, which would fall into math and algos. Could you give another thought?
Rest of the comments are fine.

Reply:
sounds good